### PR TITLE
Update blockchain node health checks

### PR DESCRIPTION
**Pull request by VladStarr**

### CODE REVIEW

File: dysnix/geth/templates/scripts/_readiness.tpl [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+set -eo pipefail
+
+HTTP_PORT="8545"
+AGE_THRESHOLD=$1
+
+if [ -z "$AGE_THRESHOLD" ]; then
+    echo "Usage: $0 <block age threshold>"; exit 1
+fi
+
+get_block_timestamp() {
```
Comment (voron) [inline dysnix/geth/templates/scripts/_readiness.tpl:16]: WDYT on adding a comment with expected function output ?

File: dysnix/geth/templates/scripts/_readiness.tpl [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    wget "http://localhost:$HTTP_PORT" -qO- \
+        --header 'Content-Type: application/json' \
+        --post-data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["latest", false],"id":1}' \
+    | grep -oE '"timestamp":".*"' | cut -d',' -f1 | cut -d':' -f2 | tr -d '"'
+}
+
+age=$(($(date +%s) - $(get_block_timestamp)))
```
Comment (voron) [inline dysnix/geth/templates/scripts/_readiness.tpl:23]: WDYT on adding some comment here to explain what happens inside?

File: dysnix/geth/templates/scripts/_readiness.tpl [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+if [ -z "$AGE_THRESHOLD" ]; then
+    echo "Usage: $0 <block age threshold>"; exit 1
+fi
+
+get_block_timestamp() {
+    wget "http://localhost:$HTTP_PORT" -qO- \
+        --header 'Content-Type: application/json' \
+        --post-data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["latest", false],"id":1}' \
+    | grep -oE '"timestamp":".*"' | cut -d',' -f1 | cut -d':' -f2 | tr -d '"'
```
Comment (voron) [inline dysnix/geth/templates/scripts/_readiness.tpl:20]: WDYT on `grep -oE '"timestamp":"[^\"]+"'` to get rid of 1'st `cut` ?

**Review by voron — APPROVED** [review]
LGTM, just some minor questions.

File: dysnix/geth/templates/scripts/_readiness.tpl (hunk 3 above)
Comment (VladStarr) [reply dysnix/geth/templates/scripts/_readiness.tpl]: Moved to `sed` substitution

File: dysnix/geth/templates/scripts/_liveness.tpl [hunk 4]
```diff
@@ -14,13 +14,16 @@ if [ -z "${AGE_THRESHOLD}" ] || [ -z "${STATE_FILE}" ]; then
...
 fi
 
+
+# expected output format: 0x50938d
 get_block_number() {
     wget "http://localhost:$HTTP_PORT" -qO- \
         --header 'Content-Type: application/json' \
         --post-data '{"jsonrpc":"2.0","method":"eth_blockNumber","id":1}' \
-    | grep -oE '"result":".*"' | tr -d '"' | cut -d':' -f2
+    | sed -r 's/.*"result":"([^"]+)".*/\1/g'
```
Comment (voron) [inline dysnix/geth/templates/scripts/_liveness.tpl:23]: who needs `jq` when we have `sed` :D

**Review by voron — APPROVED** [review]

**Pull request merged**